infrastructure/cosmosapp/httpclient.go

```diff
@@ -656,6 +657,86 @@ func (client *HTTPClient) ProposalTally(id string) (cosmosapp_interface.Tally, e
 	return tallyResp.Tally, nil
 }
 
+func (client *HTTPClient) Txs(hash string) (*model.Tx, error) {
```

Comment:
I have a small question here. While the method is named by following the RESTful API path name, but the method of `Txs` seems to imply it can be used to query multiple transaction.

Do we want to keep follow the API path or we should give the method a more meaningful name.

Reply:
I will suggest `Tx`